projects/ngx-d3timeline/src/lib/view/view.ts

```diff
@@ -1,18 +1,19 @@
+export const margin = 50;
```

Comment:
I preferred this inside View, does it have to be moved out?

Margin is intrinsic to the view, also allows it to be changed ie: what if they have long/short labels etc.